src/runtime.rs

```diff
@@ -116,6 +115,15 @@ impl Runtime {
 
                         // Host path is the absolute path to the file
                         let host_path = self.manifest.absolute_from(host_path);
+                        // If the file among list of files to be excluded, skip it
```

Comment:
`host_path` is an absolute path but I think the paths in `exclude_files` are relative to the manifest. Have you run this code to see if it works? I don't think we have tests for the runtime, but it would be really nice if we did.

Reply:
I couldn't run it because I keep running into the type mismatch error.

Looking at the sample provided for what an exclude_files can be like it looks to be in form of relative path. I would used relative path instead.